### DISCUSSION TITLE
I want to make your addon as part of Godot 4.0

### DISCUSSION THREAD

**Discussion by warriormaster12** [kagenash1/godot-behavior-tree#31] [General]
I'm currently working on a C++ version of your behavior tree that I want to merge with Godot 4.0 and I also want to add a bit more functionality like a visual editor to it. 

I was wondering if I can get a greenlight from you so that I can continue to work on the feature. In addition, if you are interested to help me out with this, let me know. 

Best regards, 
Alexander Streng

**Comment by warriormaster12** [comment]
https://github.com/warriormaster12/godot/tree/master/scene

**Comment by kagenash1** [comment]
Sounds like a good idea. I haven't worked on this plugin for a long time so feel free to take it apart.

**Reply by warriormaster12** [reply]
Thanks, will then continue to work on the feature.

**Comment by MadFlyFish** [comment]
> Sounds like a good idea. I haven't worked on this plugin for a long time so feel free to take it apart. 

Hey my friend，what are you up to recently😀。

**Comment by warriormaster12** [comment]
@kagenash1 Hi, just wanted to let you know that I've somewhat managed to port bt_task, bt_sequencer and bt_selector to C++ for Godot 4.

https://github.com/warriormaster12/godot

Requires a lot of testing though :)
https://github.com/warriormaster12/godot/releases
